src/layouts/views/WebcamView/WebcamView.js

```diff
@@ -145,13 +149,13 @@ class WebcamView extends React.Component {
         text: translations.button_retake,
       },
     };
-
+    const urlCreator = window.URL || window.webkitURL;
     return (
       <div>
         <Grid container justify="center">
           <Grid item xs={12} sm={10} md={9} className={classes.root} data-role="cameraPreview">
             <img
-              src={scans[currentStep][component].value}
+              src={urlCreator.createObjectURL(scans[currentStep][component].value)}
```

Comment:
can you move `urlCreator.createObjectURL(scans[currentStep][component].value)` to a const? looks a little bit loaded to a mark up.

Reply:
changed in `review changes` commit below